config/components/certmanager/certificate-metrics.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   name: metrics-certs  # this name should match the one appeared in kustomizeconfig.yaml
   namespace: system
 spec:
-  commonName: kueue-metrics
+  commonName: metrics
```

Comment:
Can we preserve this just in case some people are installing certs via kustomize.

Reply:
Added transformer to add prefix on the Kustomize side.